EMS/core-bundle/src/Command/Submission/ExportConfig.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Command\Submission;
+
+use EMS\Helpers\Standard\Json;
+
+final readonly class ExportConfig
+{
+    public function __construct(
+        /** @var mixed[] */
+        public array $columns,
+        public ?string $filter,
+        public ?string $filename,
+        /** @var string[] */
+        public array $emailsTo,
+        public string $subject,
+        public ?string $format,
+    ) {
+    }
+
+    public static function fromJson(string $json): self
+    {
+        $data = Json::decode($json);
+
+        if (!isset($data['columns'], $data['emails-to'], $data['subject'])) {
+            throw new \InvalidArgumentException('Invalid config JSON: missing required fields');
+        }
```

Comment:
I would use the optionResolver

https://github.com/ems-project/elasticms/blob/HEAD/elasticms-cli/src/Client/File/ImportConfig.php#L9